src/executeQuery.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+const sql = require('./sql')
+const Query = require('./query')
+const { calculateClassBreaks, calculateUniqueValue } = require('./generateBreaks/breaks')
+const _ = require('lodash')
+
+function breaksQuery (features, query, options) {
+  // TODO: add check if query is valid (or should this have been handled in query?)
```

Comment:
I don't think we need this. We also may not need the check on the next line either.